pages/task/schema/confirm.js

```diff
@@ -1,10 +1,21 @@
+const { get } = require('lodash');
+const { getAwerbQuestion } = require('../../project-version/update/submit/schema');
+
 module.exports = (task, chosenStatus) => {
   const commentRequired = stepId => {
     const nextStep = task.nextSteps.find(nextStep => nextStep.id === stepId);
     return nextStep && nextStep.commentRequired;
   };
 
-  return {
+  const awerbRequired = task => {
+    // awerb question is required for project applications / amendments that haven't yet been marked as awerbed
+    const model = get(task, 'data.model');
+    const status = get(task, 'status');
+    const wasAwerbed = get(task, 'data.meta.awerb') === 'Yes';
+    return model === 'project' && status === 'awaiting-endorsement' && !wasAwerbed;
```

Comment:
Why might a project task require endorsement but already have been awerbed?

Reply:
This schema is shared by all the task confirm pages, and we only want to show the Awerb question when a project is being endorsed.